docs/source/transaction_family_specifications/supplychain_transaction_family.rst

```diff
@@ -0,0 +1,317 @@
+***************************************
```

Comment:
Split this out to a separate PR, so the code can be validated against, once all the changes have been made to the document.